x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/WaitForRolloverReadyStep.java

```diff
@@ -53,6 +55,12 @@ public void evaluateCondition(IndexMetaData indexMetaData, Listener listener) {
             return;
         }
 
+        if (indexMetaData.getSettings().getAsBoolean(LIFECYCLE_ROLLOVER_SKIP_ROLLED, true) &&
+            indexMetaData.getRolloverInfos().get(rolloverAlias) != null) {
+            logger.trace("Index {} was already rolled, skipping", indexMetaData.getIndex().getName());
```

Comment:
1) I think this should be `info` (rather than `trace)
2) Use brackets around index name, and don't capitalize, to fit better with the typical log format
3) Nitpicky, but I'd use a message more like `index [{}] was already rolled over for alias [{}], not attempting to roll over again`